### PR TITLE
Pull Request for Issue2151: Xspress3 detector bad first point

### PR DESCRIPTION
**Pull request by helfrij**
- Removed the extra detector trigger that was added to scan initialization in #2043. 
- Modified AMXspress3XRFDetector. Instead of listening to the spectra controls' `valueChanged(double)` signal and setting the acquisition state accordingly, the detector now listens to the data source signal source.

### CODE REVIEW

File: source/beamline/BioXAS/BioXASBeamline.cpp [hunk 1]
```diff
@@ -138,6 +135,9 @@ AMAction3* BioXASBeamline::createScanInitializationAction(AMGenericStepScanConfi
 			if (braggMotor) {
 				monoInitialization = new AMListAction3(new AMListActionInfo3("BioXAS SSRL monochromator initialization", "BioXAS SSRL monochromator initialization"));
 				monoInitialization->addSubAction(braggMotor->createPowerAction(CLSMAXvMotor::PowerOn));
+
+//				monoInitialization->addSubAction(AMActionSupport::buildControlMoveAction(mono->energy(), double(configuration->scanAxisAt(0)->regionAt(0)->regionStart()) - 200));
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASBeamline.cpp]: Hey @helfrij, we can ditch these comments.

File: source/beamline/BioXAS/BioXASBeamline.cpp [hunk 2]
```diff
@@ -107,15 +107,12 @@ AMAction3* BioXASBeamline::createScanInitializationAction(AMGenericStepScanConfi
 
 					AMListAction3 *geDetectorInitialization = new AMListAction3(new AMListActionInfo3("BioXAS Xpress3 initialization", "BioXAS Xpress3 initialization"));
 					geDetectorInitialization->addSubAction(geDetector->createDisarmAction());
-					geDetectorInitialization->addSubAction(geDetector->createFramesPerAcquisitionAction(int(configuration->scanAxisAt(0)->numberOfPoints()*1.1)));	// Adding 10% just because.
+					geDetectorInitialization->addSubAction(geDetector->createFramesPerAcquisitionAction(int(configuration->scanAxisAt(0)->numberOfPoints()*1.1+2)));	// Adding 10% just because.
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASBeamline.cpp]: And we don't need that +2 anymore ;)

**Comment by helfrij** [comment]
Side will be available for testing tomorrow morning, with a glass sample similar to the ones used for testing on Main earlier.

**Comment by helfrij** [comment]
Testing on Side and Main is complete.

@XuetaoLiu-CLS, would you review?

**Comment by XuetaoLiu-CLS** [comment]
Looks good.

Time Estimate: 2 minutes

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 1 minute

**Comment by helfrij** [comment]
Thanks @XuetaoLiu-CLS @dretrex.

**Pull request merged**